claim/README.md

```diff
@@ -41,44 +38,68 @@ There are two important notes regarding connecting nodes:
 
 ## How to connect a node
 
+There will be two main flows from where you might want to connect a node to Netdata Cloud.
+* when you are on an [empty War Room](#empty-war-room) and you want to connect your first node
+* when you are at the [Manage Space](#manage-space) area and you select **Connect Nodes** to connect a node, coming from Manage Space or Manage War Room
+
+### Empty War Room
+
+#TODO: add screenshot
+
+Either at your first sign in or following ones, when you enter Netdata Cloud and are at a War Room that doesn’t have any node added to it you will be able to:
+* connect a new node to Netdata Cloud and add it to the War Room you are in
+* add a previously connected node to the War Room you are in
+
+If your case is to connect a new node and add it to the War Room, you will need to tell us what environment the node is running on (Linux, Docker, macOS, Kubernetes) and then we will provide you with a script to initiate the connection process. You just will need to copy and paste it into your node's terminal. See one of the following sections depending on your case:
+* [Linux](#connect-an-agent-running-in-linux)
+* [Docker](#connect-an-agent-running-in-docker)
+* [macOS](#connect-an-agent-running-in-macos)
+* [Kubernetes](#connect-a-kubernetes-clusters-parent-netdata-pod)
+
+Repeat this process with every node you want to add to Netdata Cloud during onboarding. You can also add more nodes once you've
+finished onboarding.
+### Manage Space
+
 To connect a node, select which War Rooms you want to add this node to with the dropdown, then copy and paste the script
-given by Cloud into your node's terminal. Hit **Enter**.
+given by Netdata Cloud into your node's terminal.
 
-```bash
-sudo netdata-claim.sh -token=TOKEN -rooms=ROOM1,ROOM2 -url=https://app.netdata.cloud
+### Connect an agent running in Linux
+
+If you want to connect a node that is running on a Linux environment, the script that will be provided to you by Netdata Cloud is the [kickstart](/packaging/installer/#automatic-one-line-installation-script) which will install the Netdata Agent on your node, if it isn't already installed, and connect the node to Netdata Cloud. It should be similar to:
+
+```
+bash <(curl -Ss https://my-netdata.io/kickstart.sh) --claim-token TOKEN --claim-rooms ROOM1,ROOM2 --claim-url https://app.netdata.cloud
```

Comment:
@hugovalente-pm instead of TOKEN, ROOM1, ROOM2 I would add some sample values.

Reply:
In this case I did not change what was the original one (examples https://learn.netdata.cloud/docs/agent/packaging/installer/methods/kickstart#connect-node-to-netdata-cloud-during-installation and https://learn.netdata.cloud/docs/agent/claim#how-to-connect-a-node) do you mean to change to some concrete example across all docs?

Reply:
@manos-saratsis won't change this for now, if we want we can revisit in a next iteration